framework/src/application/network/schema.ts

```diff
@@ -31,13 +39,21 @@ export const customPeerInfoSchema = {
 	$id: '/peerInfo/custom',
 	type: 'object',
 	properties: {
+		height: {
```

Comment:
I think this can be removed

Reply:
do we allow users to send custom properties to each other along with ipAddress and port? in that case we can remove peerInfo and only use one customSchema that will be used for `nodeInfo`

Reply:
I think it shouldnt return anything other than the ip/port for the `getPeers` endpoint because it will be untrusted/outdated information